auth0/internal/wiremock/client.go

```diff
@@ -3,88 +3,104 @@ package wiremock
 import (
 	"context"
 	"fmt"
+	"log"
 	"net/http"
 	"net/url"
 
 	"github.com/auth0/go-auth0/management"
 	"github.com/hashicorp/go-multierror"
-	"github.com/hashicorp/go-retryablehttp"
 )
 
-// Client provides communciation with the admin API of a WireMock server
-// instance.
+const (
+	resetMappingsURI  = "/__admin/mappings/reset"
+	resetScenariosURI = "/__admin/scenarios/reset"
+)
+
+// Client provides communication with the
+// admin API of a WireMock server instance.
 type Client struct {
-	host string
-	cl   *retryablehttp.Client
```

Comment:
The main reason why we're removing this is because IMO it is unnecessary to have a client with a retry mechanism and introduce another 3rd party package for this. We expect the wiremock server to always be up and running locally and for us never to have the need to retry failed requests. They should fail fast if they do and we should fix whatever the error is. :)